backend/api/urls.py

```diff
@@ -2,11 +2,19 @@
 from rest_framework_simplejwt.views import TokenObtainPairView
 from api.views.CustomTokenRefreshView import CustomTokenRefreshView
 from api.views.LogoutView import LogoutView
+from api.views.SiteView import SiteViewSet
+
+from rest_framework.routers import SimpleRouter
+
+router = SimpleRouter()
+router.register(r'sites', SiteViewSet, basename='sites')
```

Comment:
For naming endpoints, it's a good practice to use singular nouns. I would suggests to use "site" instead of sites.

This would also align with the name used for the view and the model.